scripts/remote/README.md

```diff
@@ -102,20 +112,43 @@ powerful) machine you want to run the script from that point on. Copy
 worker.
 
 
+## Command Modules
+[cm]: #command-modules
+
+`commands/default.js` simply runs the worker script without waiting
+for it to complete.
+
+`commands/correctness.js` assumes the worker script is
+`workers/run-ci.sh`, or a script based on it. If the controller
+detects relevant output, it downloads a `reports` directory from the
+remote host. *This is for backwards compatibility to make sure reports
+are visible to the NGINX server using `scripts/pash-nginx.conf`*.
```

Comment:
Idea: would it make sense to store this in the config? e.g., we could have two entries for correctness and performance reports, rather than hardcoding it in the command.

Reply:
Update: We're already changing the nginx config to point to different processes, so feel free to break compatibility here too:-)

Reply:
I'm not sure what you mean here. Store what in the config?

Reply:
I meant the same "this" as the one in the comment on line 124, which I assumed to mean "the directory where the reports go". And I meant (i) this could be part of the config, (ii) we can easily change where `pash-nginx.conf` looks for them.